AndroidSDKTests/src/test/java/com/leanplum/LeanplumTest.java

```diff
@@ -1267,4 +1273,62 @@ public void testGetDeviceId() {
     deviceId = Leanplum.getDeviceId();
     assertNotNull(deviceId);
   }
+
+  /**
+   * Tests for parsing counters
+   */
+  @Test
+  public void testStartResponseShouldParseCounters() {
+    // Setup sdk.
+    setupSDK(mContext, "/responses/simple_start_response.json");
+
+    // check that incrementing counters work
+    CountAggregator.INSTANCE.incrementCount("testCounter1");
+    assertEquals(1, CountAggregator.INSTANCE.getCounts().get("testCounter1").intValue());
+    CountAggregator.INSTANCE.incrementCount("testCounter2");
+    assertEquals(1, CountAggregator.INSTANCE.getCounts().get("testCounter2").intValue());
+  }
+
+  @Test
+  public void testParseEmptySdkCounters() throws JSONException {
+    JSONObject response = new JSONObject();
+    HashSet<String> parsedCounters = Leanplum.parseSdkCounters(response);
+    assertEquals(new HashSet<String>(), parsedCounters);
+  }
+
+  @Test
+  public void testParseSdkCounters() throws JSONException {
+    JSONObject response = new JSONObject();
+    response.put(Constants.Keys.ENABLED_COUNTERS, new JSONArray("[\"test\"]"));
+    HashSet<String> parsedCounters = Leanplum.parseSdkCounters(response);
+    assertEquals(new HashSet<String>(Arrays.asList("test")), parsedCounters);
+  }
+
+
+  /**
+   * Tests for parsing feature flags
+   */
+  @Test
+  public void testStartResponseShouldParseFeatureFlags() {
+    // Setup sdk.
+    setupSDK(mContext, "/responses/simple_start_response.json");
+
+    assertEquals(true, FeatureFlagManager.INSTANCE.isFeatureFlagEnabled("testFeatureFlag1"));
+    assertEquals(true, FeatureFlagManager.INSTANCE.isFeatureFlagEnabled("testFeatureFlag2"));
+  }
```

Comment:
let's add a line - this ensures that we aren't returning true for everything
`assertEquals(false, FeatureFlagManager.INSTANCE.isFeatureFlagEnabled("missingFeatureFlag"));`